### ISSUE TITLE
Scaled ReLU Matters for Training Vision Transformers【畳み込みを用いたViTの学習の安定】

### ISSUE THREAD

**Issue by SyunkiTakase** [SyunkiTakase/Paper-Research#48]
## 概要
Vision Transformer(ViT)は学習率，Optimizer，WarmUp Epochなどのパラメータに敏感であるため，学習が難しい．そこでこの論文ではパッチ分割の代わりにReLUが内部に入っている畳み込みシステム(Conv-Stem)を導入することで学習の安定性を向上させている．
この手法では入力画像をパッチに分割せずにConv-Stemに入力し，特徴マップを抽出してからTransformer Encoderに入力している．これにより，入力画像のパッチ化の手間を省き，Conv-Stemによって低レベルの特徴を抽出可能である．

学習率，Optimizer，WarmUp Epochを変更したときのパッチ分割したViT(ViTp)とConv-Stemを使用したViT(ViTc)のImageNetの分類精度を比較している．
![image](https://github.com/SyunkiTakase/Paper-Research/assets/161103101/7d84d046-798d-4b4a-ac72-60305618c470)

学習率，Optimizer，WarmUp Epoch，モデル，Conv-Stemを変更したときのViTcのImageNetの分類精度を比較している．
![image](https://github.com/SyunkiTakase/Paper-Research/assets/161103101/70f2114b-48fe-486c-8609-5e3cfb3ce78c)

結果としては，(畳み込み → Batch Normalization → ReLU) × 3 → 畳み込み → Layer Normalizationが最も良い性能となっている．

## 論文リンク
https://arxiv.org/abs/2109.03810

## 著者/発表学会/年
Wang, Pichao, et al. "Scaled relu matters for training vision transformers." Proceedings of the AAAI Conference on Artificial Intelligence. Vol. 36. No. 3. 2022.

```
@inproceedings{wang2022scaled,
  title={Scaled relu matters for training vision transformers},
  author={Wang, Pichao and Wang, Xue and Luo, Hao and Zhou, Jingkai and Zhou, Zhipeng and Wang, Fan and Li, Hao and Jin, Rong},
  booktitle={Proceedings of the AAAI Conference on Artificial Intelligence},
  volume={36},
  number={3},
  pages={2495--2503},
  year={2022}
}
```

## コメント
Wavelet Vision TransformerのConv-Stemを提案した論文だったので読んだ．Conv-Stem，Optimizer，WarmUp Epoch，学習率などの変更でかなり精度が異なっておりViTにはハイパーパラメータの影響が大きいことが分かった．
